### PR TITLE
"ReadMe changes"

### PR DESCRIPTION
**Pull request by MallanagoudaB**

### CODE REVIEW

**Review by VISHNUDAS-tunerlabs — CHANGES_REQUESTED** [review]
Reviewed On Nov 11 2024

File: README.md [hunk 1]
```diff
@@ -512,207 +509,209 @@ Before setting up the following Survey application, dependencies given below sho
...
-    git clone -b main https://github.com/ELEVATE-Project/observation-survey-projects-pwa.git
-    ``` 
+      ```
+      git clone -b main https://github.com/ELEVATE-Project/samiksha-service.git && \
+      git clone -b main https://github.com/ELEVATE-Project/entity-management.git && \
+      git clone -b master https://github.com/ELEVATE-Project/user.git && \
+      git clone -b master https://github.com/ELEVATE-Project/notification.git && \
+      git clone -b main https://github.com/ELEVATE-Project/interface-service.git && \
+      git clone -b master https://github.com/ELEVATE-Project/scheduler.git && \
+      git clone -b main https://github.com/ELEVATE-Project/observation-survey-projects-pwa.git
```
[the remark this answers is not in the document]
Comment (MallanagoudaB) [reply README.md:525]: but that will be merged today only

File: documentation/1.0.0/native/envs/interface_env [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
-SCHEDULER_SERVICE_BASE_URL=http://localhost:4000
-SUPPORTED_HTTP_TYPES="GET POST PUT PATCH DELETE"
-USER_SERVICE_BASE_URL=http://localhost:3001
-ALLOWED_HOST=http://localhost:8100
-PROJECT_SERVICE_BASE_URL=http://localhost:5000
+API_DOC_URL= /interface/api-doc
+APPLICATION_ENV= development
+APPLICATION_PORT= 3569
+ENTITY_SERVICE_BASE_URL= http://localhost:5001
+INSTALLED_PACKAGES= elevate-user elevate-mentoring elevate-scheduler elevate-entity-management elevate-self-creation-portal elevate-survey
```
[the remark this answers is not in the document]
Comment (MallanagoudaB) [reply documentation/1.0.0/native/envs/interface_env:5]: @VISHNUDAS-tunerlabs  i have resmoved all those things dont know why its not refelcting here

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (MallanagoudaB) [reply README.md:525]: just confirmed merged to main branch with amogh

**Review by VISHNUDAS-tunerlabs — APPROVED** [review]

**Pull request merged**